app/src/main/java/ca/bc/gov/health/ircreader/ui/scanresult/BarcodeScanResultFragment.kt

```diff
@@ -1,20 +1,100 @@
 package ca.bc.gov.health.ircreader.ui.scanresult
 
+import android.graphics.Color
+import android.os.Binder
 import android.os.Bundle
+import android.view.LayoutInflater
 import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.core.content.ContextCompat
 import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.NavHostFragment
 import ca.bc.gov.health.ircreader.R
-import dagger.hilt.android.AndroidEntryPoint
 
 /**
  * [BarcodeScanResultFragment]
  *
  * @author Pinakin Kansara
  */
-@AndroidEntryPoint
+
 class BarcodeScanResultFragment : Fragment(R.layout.fragment_barcode_scan_result) {
 
+    /*
+    * Both of these variable control the UI for status screen
+    * vaccinationStatus = 0 is "No records found"
+    * vaccinationStatus = 1 is "Partially vaccinated"
+    * vaccinationStatus = 2 is "Fully vaccinated"
+    * */
+    private var userName : String = ""
+    private var vaccinationStatus : Int = 0
```

Comment:
All this logic should go to view model and we have to observe the data to reflect the changes in the fragment.